easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
W.r.t. timing: no pressure there, we still have loads of time to get PGI support included for the next EB release (planned for end of Oct'15).